.goreleaser.yml

```
@@ -17,10 +18,42 @@ builds:
   - -s -w -extldflags "-static" -X github.com/shipwright-io/cli/pkg/shp/cmd/version.version={{.Version}}
   main: ./cmd/shp/main.go
   binary: shp
+  id: shp
+- env:
+    - CGO_ENABLED=0
+  goos:
+    - linux
+    - darwin
+    - windows
+  goarch:
+    - amd64
+    - arm64
+  flags:
+    - -trimpath
+  ldflags:
+    - -s -w -extldflags "-static" -X github.com/shipwright-io/cli/pkg/shp/cmd/version.version={{.Version}}
+  main: ./cmd/shp/main.go
+  binary: kubectl-shp
+  id: kubectl-shp
```

Just some thoughts here: The `id` should come first in my opinion as it severs mostly as an internal identifier and makes for better readability to start the list entry with its identifier.

Also, the new entry used a different indentation compared to the old one. It does not matter technically, but I think we should try to keep the same style within one YAML file.
```suggestion
- id: kubectl-shp
  env:
  - CGO_ENABLED=0
  goos:
  - linux
  - darwin
  - windows
  goarch:
  - amd64
  - arm64
  flags:
  - -trimpath
  ldflags:
  - -s -w -extldflags "-static" -X github.com/shipwright-io/cli/pkg/shp/cmd/version.version={{.Version}}
  main: ./cmd/shp/main.go
  binary: kubectl-shp

```

---

@HeavyWombat can u re-review pls?